acceptance/bundle/telemetry/test.toml

```diff
@@ -3,7 +3,7 @@ IncludeRequestHeaders = ["User-Agent"]
 
 [[Repls]]
 Old = '"execution_time_ms": \d{1,5},'
-New = '"execution_time_ms": SMALL_INT,'
+New = '"execution_time_ms": "SMALL_INT",'
```

Comment:
The quotes are added so that even after replacement the file remains a valid JSON.

Reply:
Yeah, you are right. Reverted.